android/app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,7 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
@enochtsang is this okay? what were you saying again about permissions and deploying the app on the playstore?

Reply:
Some permissions will need special requirements on the play store, like a privacy policy. I don't know if this will affect it. But this is not the one that was affecting it.